test/handlers/assembly-documentation/amd64-tests.js

```diff
@@ -65,16 +65,6 @@ describe('amd64 assembly documentation', () => {
           }).catch(e => { throw e; });
    });
 
-    it('should respond to json for unknown opcodes', () => {
-        return chai.request(app)
-            .get('/asm/NOANOPCODE')
-            .set('Accept', 'application/json')
-            .then(res => {
-                res.should.have.status(404);
-                res.should.be.json;
-            }).catch(e => { throw e; });
-    });
```

Comment:
Obsolete test case, the existing test case that looks like this already checks this: 
```js
    it('returns 404 for unknown opcodes', () => {
        return chai.request(app).get('/asm/mov_oh_wait')
            .then(res => {
                res.should.have.status(404);
                res.should.be.json;
                res.body.should.deep.equal({ error: 'Unknown opcode \'MOV_OH_WAIT\'' });
            }).catch(e => { throw e; });
    });
```